### DISCUSSION TITLE
不知道为啥今天打开 之前解锁成功的游戏都不能玩了 重新入库后还是购买选项。。

### DISCUSSION THREAD

**Discussion by karson0014** [pjy612/SteamManifestCache#25] [Question / Answer]
如题

**Comment by karson0014** [comment]
卸载后 重新入库 用DB安装 显示无许可

**Comment by ZEERDEER** [comment]
请详细描述你做了什么，而不是一个“不知道”，例如：更新了steam beta

**Comment by pjy612** [comment]
首先 SAI 并不提供解锁功能。
其次，盲猜您应该用的是 SteamTools。
那么请先看看 wiki 。确认下 tools 的解锁模式是否正常开启。
然后通过 tools 启动Steam ，看悬浮窗颜色是否正确。
确保能正常使用 解锁模式 之后，再尝试用工具或自行手动入库。

另外有必要可以根据 TG 群组中 的管理员 回复，重装一下 tools。
